libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/infopanel/InfoPanelHeaderActiveGuidanceBinder.kt

```diff
@@ -17,12 +16,9 @@ internal class InfoPanelHeaderActiveGuidanceBinder(
 ) : UIBinder {
 
     override fun bind(viewGroup: ViewGroup): MapboxNavigationObserver {
-        val scene = Scene.getSceneForLayout(
-            viewGroup,
-            R.layout.mapbox_info_panel_header_active_guidance_layout,
-            viewGroup.context
-        )
-        TransitionManager.go(scene)
+        viewGroup.removeAllViews()
+        LayoutInflater.from(viewGroup.context)
+            .inflate(R.layout.mapbox_info_panel_header_active_guidance_layout, viewGroup, true)
         val binding = MapboxInfoPanelHeaderActiveGuidanceLayoutBinding.bind(viewGroup)
```

Comment:
```suggestion
        val binding = MapboxInfoPanelHeaderActiveGuidanceLayoutBinding.inflate(
            LayoutInflater.from(viewGroup.context),
            viewGroup
        )
```
Same for all other binders

Reply:
Goot catch. Forgot that `LayoutInflater.inflate(layoutId, root)` passes `attachToRoot = true` when `root` is not-null.

Reply:
Done